src/ck-ldb/LBDBManager.h

```diff
@@ -1,316 +0,0 @@
-
-  inline void LocalBarrierOn() 
-       { localBarrier.TurnOn();}
-  inline void LocalBarrierOff() 
-       { localBarrier.TurnOff();}
-
-  inline LBOM *LbOM(LDOMHandle h) 
-       { return oms[h.handle]; };
-  inline LBObj *LbObj(const LDObjHandle &h) const 
-       { return objs[h.handle].obj; };
-  inline LBObj *LbObjIdx(int h) const 
-       { return objs[h].obj; };
-  void DumpDatabase(void);
-
-  inline void TurnStatsOn(void) 
-       {statsAreOn = true; machineUtil.StatsOn();}
-  inline void TurnStatsOff(void) 
-       {statsAreOn = false;machineUtil.StatsOff();}
-  inline bool StatsOn(void) const 
-       { return statsAreOn; };
-
-  void SetupPredictor(LDPredictModelFn on, LDPredictWindowFn onWin, LDPredictFn off, LDPredictModelFn change, void* data);
-  inline void TurnPredictorOn(void *model) {
-    if (predictCBFn!=NULL) predictCBFn->on(predictCBFn->data, model);
-    else CmiPrintf("Predictor not supported in this load balancer\n");
-  }
-  inline void TurnPredictorOn(void *model, int wind) {
-    if (predictCBFn!=NULL) predictCBFn->onWin(predictCBFn->data, model, wind);
-    else CmiPrintf("Predictor not supported in this load balancer\n");
-  }
-  inline void TurnPredictorOff(void) {
-    if (predictCBFn!=NULL) predictCBFn->off(predictCBFn->data);
-    else CmiPrintf("Predictor not supported in this load balancer\n");
-  }
-  /* the parameter model is really of class LBPredictorFunction in file LBDatabase.h */
-  inline void ChangePredictor(void *model) {
-    if (predictCBFn!=NULL) predictCBFn->change(predictCBFn->data, model);
-    else CmiPrintf("Predictor not supported in this load balancer");
-  }
-
-  void Send(const LDOMHandle &destOM, const CmiUInt8 &destid, unsigned int bytes, int destObjProc);
-  void MulticastSend(const LDOMHandle &destOM, CmiUInt8 *destids, int ndests, unsigned int bytes, int nMsgs);
-  int ObjDataCount();
-  void GetObjData(LDObjData *data);
-  inline int CommDataCount() { 
-    if (commTable)
-      return commTable->CommCount();
-    else return 0;
-  }
-  inline void GetCommData(LDCommData *data) 
-       { if (commTable) commTable->GetCommData(data); };
-
-  inline void GetCommInfo(int& bytes, int& msgs, int& withinbytes, int& acrossbytes, int& num_nghbors, int& hops, int& hopbytes) {
-    if (commTable)
-      commTable->GetCommInfo(bytes, msgs, withinbytes, acrossbytes, num_nghbors, hops, hopbytes);
-  };
-
-  void MetaLBResumeWaitingChares(int lb_ideal_period);
-  void MetaLBCallLBOnChares();
-  int  Migrate(LDObjHandle h, int dest);
-  void Migrated(LDObjHandle h, int waitBarrier=1);
-  int  NotifyMigrated(LDMigratedFn fn, void* data);
-  void TurnOnNotifyMigrated(int handle)
-       { migrateCBList[handle]->on = 1; }
-  void TurnOffNotifyMigrated(int handle)
-       { migrateCBList[handle]->on = 0; }
-  void RemoveNotifyMigrated(int handle);
-
-  inline void TurnManualLBOn() 
-       { useBarrier = false; LocalBarrierOff(); }
-  inline void TurnManualLBOff() 
```

Comment:
This was removed from here, but seems to differ from what was added to LBDatabase.

Reply:
Fixed.